src/app/container/container.component.html

```diff
@@ -128,23 +128,21 @@
               Description
             </label>
             <div class="well well-sm">
-              <pre><markdown>{{ tool?.description || 'No description associated with this container.' }}</markdown></pre>
```

Comment:
FYI, tools and workflows do use markdown for the description. 
This may need to be added back eventually depending on why it was token out